### DISCUSSION TITLE
3D support

### DISCUSSION THREAD

**Discussion by riazorsound** [sumatrapdfreader/sumatrapdf#3160] [General]
Would it be possible to visualize 3D objects in the pdf? It shows a message that tells us there is a 3D content, but it is not possible to move it, or even see it.

**Comment by GitHubRulesOK** [comment]
Dynamic forms, Dynamic Models, Dynamic signatures, Flash Media and  other extended OCG X-Objects
Are generally non-standard Adobe proprietary content so not part of a basic PDF structure even basic AcroForms are problematic by their very split appearance nature. Thus unlikely that the complex 2 types of 3D would be possible in a fast viewer as highly GPU intensive. even Large teams of programmers at Foxit struggle to reverse the way Adobe built them closest is possibly Team Tracker X-Change
![image](https://user-images.githubusercontent.com/12754913/205491936-3b700be3-9cd2-4a16-a1f8-62e5bc8cd028.png)
